.agents/skills/docs-sync/SKILL.md

```
@@ -0,0 +1,157 @@
+---
+name: docs-sync
+description: Detect and close drift between code and the Docusaurus site in `website/`. Use when the user asks to sync docs with code, audit docs drift, or after adding/removing packages, CLI commands, or MCP tools. Trigger words - sync docs, docs drift, update docs, website out of date, missing docs page.
+---
+
+# docs-sync
+
+Three-level pipeline for keeping `website/` in sync with the code. Each level
+runs independently — **do not** skip level 1 before running the later ones.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)